src/clr-base/package.json

```diff
@@ -3,20 +3,23 @@
     "version": "2.0.2",
     "description": "Custom Elements for Clarity",
     "homepage": "https://clarity.design",
+    "sideEffects": [
+      "./**/*clr-wc-element.js"
```

Comment:
https://webpack.js.org/guides/tree-shaking/#mark-the-file-as-side-effect-free

When we start to add additional components we can add them like this:
https://github.com/coryrylan/lithium/blob/master/projects/lithium/package.json#L4

This will make sure random utilities is not pulled into the bundles. Example the ie ponyfills. The side effects will allow the bundler to only include everything in the given file if imported. We only want side effects in component/element imports so the element will auto register.